### ISSUE TITLE
Have partition_cols only for S3 instead of having it also in the table

### ISSUE THREAD

**Issue by Falydoor** [awslabs/aws-data-wrangler#801]
Hello,

I use https://aws-data-wrangler.readthedocs.io/en/2.9.0/stubs/awswrangler.s3.to_parquet.html#awswrangler.s3.to_parquet to write a parquet file to S3 and store it in a Glue catalog.

I use `partition_cols` to have my partitions created on S3 (ex: I have a column `file_type`) but I was wondering if it is possible to not have the column `file_type` in my Glue table?

So to resume, I want to use a column for the partitions on S3 but I don't want this column to be present in the table.

**Comment by jaidisido** [comment]
Hi @Falydoor - Glue API calls do not have an option to ignore partition columns. However it's possible to achieve the same by breaking it down into multiple steps:
1. Call `wr.s3.to_parquet` without specifying a `database` or `table` (i.e. saving to S3 only)
2. Call [wr.s3.read_parquet_metadata](https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.s3.read_parquet_metadata.html#awswrangler.s3.read_parquet_metadata) to obtain the `column_types` and `partition_types`
3. Call `wr.catalog.create_table` using the `column_types` information from the previous call (but ignoring the `partition_types` information).
In short, the idea is to save to S3 without the partitions but create a Glue table definition on top without them

**Comment by Falydoor** [comment]
Thank you @jaidisido !
